README.md

```diff
@@ -55,11 +61,38 @@ Install Cachix from master:
     $ nix-env -if https://github.com/cachix/cachix/tarball/master --substituters 'https://cache.nixos.org https://cachix.cachix.org' --trusted-public-keys 'cachix.cachix.org-1:eWNHQldwUO7G2VkjpnjDbWwy4KQ/HNxht7H4SSoMckM= cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY='
 ```
 
-Or with Nix 2.4+:
 
-```
-    $ nix profile install github:cachix/cachix/latest
-```
+### Dev setup for VS Code
+
+1. [Enable flakes](https://nixos.wiki/wiki/Flakes#Enable_flakes)
+
+2. Install [direnv](https://direnv.net/)
```

Comment:
This needs to explain exactly how to do it

Reply:
fixed 77f9d53